### ISSUE TITLE
Chrome 109 renamed an option, which causes tests to not use the newer headless mode

### ISSUE THREAD

**Issue by mdmintz** [seleniumbase/SeleniumBase#1670]
### Chrome 109 renamed an option, which causes tests to not use the newer headless mode

For Chrome 96 through Chrome 108, ``--headless=chrome`` activated the new headless mode.

For Chrome 109 and above, ``--headless=new`` must be used to activate the new headless mode.

To fix this, the browser version must be detected first. Then, if the Chrome version is between 96 and 108, use this:

```python
options.add_argument("--headless=chrome")
```

And if the detected Chrome version is 109 or newer, then use this:

```python
options.add_argument("--headless=new")
```

(NOTE: SeleniumBase triggers the new headless mode by adding ``--headless2`` as a ``pytest`` command-line option.)

**Comment by mdmintz** [comment]
The temporary workaround for SeleniumBase tests until this issue is fixed:

```bash
pytest --chromium-arg="--headless=new"
```

**Comment by mdmintz** [comment]
Released in ``4.11.4`` - https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.11.4
